crates/bevy_render/src/render_resource/texture.rs

```diff
@@ -7,8 +7,22 @@ pub struct TextureId(Uuid);
 
 /// A GPU-accessible texture.
 ///
-/// May be converted from and dereferences to a wgpu [`Texture`](wgpu::Texture).
-/// Can be created via [`RenderDevice::create_texture`](crate::renderer::RenderDevice::create_texture).
+/// Although primarily used to to store data that "texture" the surfaces of solids, textures can also be used as general purpose
+/// storage for data of 1, 2, or 3 dimensional data.
+///
+/// It is created by setting up a [`TextureDescriptor`](crate::render_resource::TextureDescriptor)
+/// and then calling [`RenderDevice::create_texture`](crate::renderer::RenderDevice::create_texture) or  
+/// [`RenderDevice::create_texture_with_data`](crate::renderer::RenderDevice::create_texture_with_data).
+///
+/// Note that a closely related data structure is [`Image`](crate::texture::Image). It can be thought of a CPU-side analogue of
```

Comment:
Semantic line breaks

Reply:
@alice-i-cecile hmm, can you clarify this?

Reply:
Split your lines at the sentence or clause level when convenient to make them easier to read and review :)